app/guid-node/registrations/template.hbs

```diff
@@ -0,0 +1 @@
+{{node-navbar node=model.taskInstance.value active='registrations'}}
```

Comment:
Where's the model?

Reply:
It seems to come from the guid-node's route, which is kinda weird now that you mention it. I will be removing this route as per your comment though.

Reply:
Although it seem's like it would be an unneeded amount of work to properly set up a dummy app to test this, provided that we wont use until one of these pages are implemented.